lib/python/treadmill_aws/bootstrap/node/aws.linux/bin/parts/99_remove_cgroups.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+AWK="{{ _alias.awk }}"
+CAT="{{ _alias.cat }}"
+ECHO="{{ _alias.echo }}"
+GREP="{{ _alias.grep }}"
+
+CGROUP_BASE="/sys/fs/cgroup"
+TREADMILL_ROOT_CGROUP=treadmill
+
+# get total physical memory in Bytes
+PHYSICAL_MEMORY_KB="$(${GREP} MemTotal /proc/meminfo | ${AWK} '{print $2}')"
+PHYSICAL_MEMORY="$((1024*$PHYSICAL_MEMORY_KB))"
+
+# Get total number of CPU cores
+CPUSET_ALL_CORES="$(${CAT} ${CGROUP_BASE}/cpuset/cpuset.cpus)"
+
+# Reset CPU limits
+${ECHO} 100 >${CGROUP_BASE}/cpu/system.slice/cpu.shares
+${ECHO} 100 >${CGROUP_BASE}/cpu/${TREADMILL_ROOT_CGROUP}/cpu.shares
+
+# Reset CPU core assignment
+${ECHO} ${CPUSET_ALL_CORES} >${CGROUP_BASE}/cpuset/system.slice/cpuset.cpus
+${ECHO} ${CPUSET_ALL_CORES} >${CGROUP_BASE}/cpuset/${TREADMILL_ROOT_CGROUP}/cpuset.cpus
+
+# Reset memory limits
+${ECHO} ${PHYSICAL_MEMORY} >${CGROUP_BASE}/memory/system.slice/memory.limit_in_bytes
```

Comment:
i think you can echo -1 to unset the limit...

also, make sure you fix swap limit too, and there is particular order you have to do it i think - check the docs. 

another (probably more robust way) is to echo whatever value is in the very root cgroup:

cat /sys/fs/cgroup/memory/memory.limit_in_bytes

same for cpuset.

Reply:
This is the very root cgroup (CGROUP_BASE = /sys/fs/cgroup) so using it as a reference would not work. 

cpuset.cpus will not accept a -1 as an argument.

Reply:
for memory i think it is easier to put '-1'